version/constants.go

```diff
@@ -109,6 +109,12 @@ var (
 		constants.MainnetID: time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
 		constants.FujiID:    time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
 	}
+
+	EForkTimes = map[uint32]time.Time{
+		constants.MainnetID: time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+		constants.FujiID:    time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+	}
+	TempForkTime = time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Why are we treating the E upgrade differently than the prior upgrades? Normally we activate all of our upgrades on custom networks.

Reply:
It's just that while in next PRs the verification logic is in place, but the tx building is not yet ready. Hence I'd switched off the E-fork even in test until I have a coherent codebase.